contrib/freshmaker-cli

```diff
@@ -141,6 +146,16 @@ def get_api_url(deployment, env):
 
     return envs[env]
 
+def get_errata_api(env):
+    ERRATA_APIS = {
+                "prod": "https://errata.engineering.redhat.com/filter/2958.json",
```

Comment:
I don't love internal URLs being hard-coded in an upstream repo, but I'll let it be since the precedent has already been set in a prior commit for this script.